.travis.yml

```diff
@@ -8,8 +8,17 @@ julia:
   - 0.4
   - 0.5
   - 0.6
```

Comment:
Thanks. Let's drop 0.4 and 0.5. Registered packages are now required to support 0.6 or higher, but not previous versions.

Reply:
Done